lib/Ceilometer/Types/Base.hs

```diff
@@ -0,0 +1,24 @@
+module Ceilometer.Types.Base where
+
+import           Control.Isomorphism.Partial.Iso
+import           Data.Bimap                      (Bimap)
+import qualified Data.Bimap                      as B
+import           Data.Word
+
+-- |Iso generator
+generateIsoFromBimap :: (Ord a, Ord b) => Bimap a b -> Iso a b
```

Comment:
We're assuming the domain is a set, yes? In this case, it is because it's already been stuffed into the bimap, deleting any duplicates.